pkg/store/verifiable/store.go

```diff
@@ -229,6 +231,62 @@ func (s *StoreImplementation) GetPresentations() ([]*Record, error) {
 	return s.getAllRecords(presentationNameDataKey(""), getPresentationName)
 }
 
+// RemoveCredentialByName removes the verifiable credential and its records containing given name.
+func (s *StoreImplementation) RemoveCredentialByName(name string) error {
+	if name == "" {
+		return errors.New("credential name is mandatory")
+	}
+
+	id, err := s.GetCredentialIDByName(name)
+	if err != nil {
+		return fmt.Errorf("get credential id using name : %w", err)
+	}
+
+	err = s.remove(id, credentialNameDataKey(name))
+	if err != nil {
+		return fmt.Errorf("unable to delete credential : %w", err)
+	}
+
+	return nil
+}
+
+// RemovePresentationByName removes the verifiable presentation and its records containing given name.
+func (s *StoreImplementation) RemovePresentationByName(name string) error {
+	if name == "" {
+		return errors.New("presentation name is mandatory")
+	}
+
+	id, err := s.GetPresentationIDByName(name)
+	if err != nil {
+		return fmt.Errorf("get presentation id using name : %w", err)
+	}
+
+	err = s.remove(id, presentationNameDataKey(name))
+	if err != nil {
+		return fmt.Errorf("unable to delete presentation : %w", err)
+	}
+
+	return nil
+}
+
+func (s *StoreImplementation) remove(id, recordKey string) error {
+	if id == "" {
+		return errors.New("id is required")
+	}
+
```

Comment:
nit: It will never happen, I would remove it.

Reply:
Done.

Reply:
Can you please merge PR if everything is ok?

Reply:
I can't :) @troyronda @fqutishat @llorllale ^